src/components/layout/Layout.tsx

```diff
@@ -7,7 +7,7 @@ export const Layout = (props: LayoutProps): ReactElement => {
   return (
     <>
       <MainNavigation />
-      <main className="max-w-xl mx-auto bg-white rounded-xl  overflow-hidden md:max-w-2xl">
+      <main className="max-w-xl mx-auto bg-white rounded-xl  overflow-hidden md:max-w-6xl">
         {props.children}
```

Comment:
I recommend destructuring props instead of accessing it like this. 

`{ children }  : LayoutProps ` looks better and is more understandable than ` props: LayoutProps `